Flixable_com_Scraper_.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>IMO, building the URL with <code>.format() </code> is preferred in this case over <code>+</code>  to introduce some minimal future proofing.  If we want to start using the other parameters (i.e. <code>min-rating</code>  or <code>max-year</code> ) this is an easy transition compared to expanding the full url with concatenation via <code>+</code> , and more importantly it will be more readable than a long string of <code>+</code>.  I'm also more in favor of declaring this as a constant at the top of the script for easier changes.  For example, I know we didn't have <code>genre/movies</code>  in the URL when we viewed this together, idk if it makes a real difference, but it would be very easy to change at the top and let the change flow through.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AdamSpannbauer/flixable_ml_dsi/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
On 2nd thought, the best way is probably to let requests  handle building the url for us instead of us doing the work with .format  or +.   It does this way better and way cleaner.



```python
url = "https://flixable.com"
params = {'min-rating': 0,
          'min-year': 0,
          'max-year': 3000,
          'page': 1}

response = requests.get(url, params=params)

# Show the URL that `requests` built
# Output: https://flixable.com/?min-rating=0&min-year=0&max-year=3000&page=1
print(response.url)
response.text
 ```